bloop.json

```diff
@@ -3,14 +3,11 @@
   "url": "https://github.com/scalacenter/bloop/releases/download/v1.4.1/bloop-coursier.json",
   "hash": "sha256:2e6a873183e5e22712913bfdab1331d0a7792167c369fee5be0c83e27572fe12",
   "depends": "coursier",
-  "bin": "bloop",
-  "env_add_path": "$dir",
+  "bin": "bloop.bat",
   "env_set": {
-    "BLOOP_HOME": "$dir",
-    "BLOOP_IN_SCOOP": "true"
+    "BLOOP_HOME": "$dir"
   },
   "installer": {
-    "script": "coursier install --install-dir $dir --default-channels=false --channel $dir bloop"
```

Comment:
What do you think of keeping these arguments to use the channel `bloop-coursier.json`? This is the channel we distribute in our release process and we use it in brew as well.